com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImpl.java

```diff
@@ -222,13 +205,21 @@ private boolean containsOnlyFinishedStates(List<Operation.State> statusList) {
 
     @Override
     public ResponseEntity<List<String>> getOperationActions(String spaceGuid, String operationId) {
-        Operation operation = operationService.createQuery()
-                                              .processId(operationId)
-                                              .singleResult();
+        Operation operation = queryOperation(operationId);
         return ResponseEntity.ok()
                              .body(getAvailableActions(operation));
     }
 
+    private Operation queryOperation(String operationId) {
+        try {
+            return operationService.createQuery()
+                .processId(operationId)
```

Comment:
Use this formatter:
https://github.com/cloudfoundry-incubator/multiapps/blob/master/ide/eclipse-formatter.xml